src/main.js

```diff
@@ -336,19 +336,35 @@ class MiniGraphCard extends LitElement {
       </div>` : '';
   }
 
+  generateLegend(entity, index) {
+    let legend = this.computeName(index);
+    const state = this.getEntityState(index);
+    const show_legend_state = this.config.entities[index];
+
+    if (show_legend_state) {
+      legend += ` (${this.computeState(state)} ${this.computeUom(index)})`;
```

Comment:
For "%" units some people may say like "do not want to see a whitespace before %" (https://github.com/home-assistant/frontend/pull/19397). Personally me do not care.

Reply:
I don't get this data in this.entity[index].attributes (tryed with system cpu usage metric)
I don't know if I can get this to work

Reply:
This `this.config.entities[index].show_legend_state` may be not defined (and thus treated as `false`) - so please consider this case.

Reply:
If null, it should be considered as false.
I've added the check just in case, but it shouldn't be necessary

Reply:
> I don't get this data in this.entity[index].attributes (tryed with system cpu usage metric)

Could you elaborate?

Reply:
I get the entity attributes in this.entity[index].attributes (here firendly_name, state_class, unit_of_measurement). I don't get any blank_before_percent attribute or something like that to detect whether I put the blank space

Reply:
> I don't get any blank_before_percent attribute or something like that to detect whether I put the blank space

In HA frontend a special function is used to either set a blank or not between a state & unit, see the mentioned PR.
But let's not bother about these things in this PR ))

Reply:
Please let me know if you done with changes.

Reply:
yeah i'm done :)